.github/workflows/build.yml

```diff
@@ -35,6 +35,8 @@ jobs:
         working-directory: rust
       - run: CROSS_CONFIG=Cross.toml cross build --release --target x86_64-unknown-linux-gnu
         working-directory: rust
+      - run: CROSS_CONFIG=Cross.toml cross build --release --target x86_64-unknown-linux-gnu --features experimental
```

Comment:
if we set it here we will ship the experimental version when we release.

Reply:
Ah, thanks. I mostly wanted to see what github would do, so the PR is not yet meant to be taken seriously (yet?)